src/superadmin.yml

```diff
@@ -8,6 +8,7 @@ admins:
   f9a1982e-252e-4ed3-92ed-52b0506a39c9:
     last_login_name: Prozza
     is_activated: true
+    cmdspy_is_on: false
```

Comment:
I dun goofed and forgot to change the config string but:
The reason I made it false it to prevent everyone from getting commandspy when they first join if the admin list is wiped.